### PR TITLE
Added avro producer for data gen for the new schema registry integrat…

### PR DESCRIPTION
**Pull request by hjafarpour**
…ion.
This PR adds a new Avro Producer that will generate sample data based on our latest avro support with schema registry integration. The default sr url is localhost:8081 but can be set as a command line parameter using `schemaregistryurl` argument.

### CODE REVIEW

File: ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java [hunk 1]
```diff
@@ -296,6 +307,9 @@ public Builder parseArg(String arg) throws IOException {
           case "maxInterval":
             maxInterval = parseIterations(argValue);
             break;
+          case "schemaregistryurl":
+            schemaregistryurl = schemaregistryurl;
```
Comment (rodesai) [inline ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java]: I think this should be argValue?

Comment (hjafarpour) [reply ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java]: Yes, updated it. Thanks.

File: ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java [hunk 2]
```diff
@@ -296,6 +307,9 @@ public Builder parseArg(String arg) throws IOException {
           case "maxInterval":
             maxInterval = parseIterations(argValue);
             break;
+          case "schemaregistryurl":
```
Comment (apurvam) [inline ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java]: Should be camel case to be consistent with the other arguments.

Comment (hjafarpour) [reply ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java]: Updated the name.

**Pull request merged**